modules/apps/commerce/commerce-service/service.xml

```diff
@@ -979,5 +979,6 @@
 		<exception>GuestCartItemMaxAllowed</exception>
 		<exception>GuestCartMaxAllowed</exception>
 		<exception>ProductBundle</exception>
+		<exception>RequiredOrderItemOption</exception>
```

Comment:
This is incorrect. We need to separate all the exceptions. Do not use single exception for all cases.

Reply:
I only created one since I figured the others could be covered either by preexisting exceptions or by the fact that we shouldn't need to throw an exception (e.g. sku contributor since we should always be able to fill in the info). We can discuss tomorrow about what you expect the exceptions to look like though.